airbyte-integrations/connectors/source-dynamodb/src/main/resources/spec.json

```diff
@@ -4,7 +4,6 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Dynamodb Source Spec",
     "type": "object",
-    "required": ["access_key_id", "secret_access_key"],
```

Comment:
In the config I'd like maybe to add a auth object where the user should select the type of access: access keys or role based

Reply:
Would this force a breaking change if the auth object is required?

Reply:
And if we are to do a breaking change, do you mean something like this in the GA connector?

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/spec.json#L10

Reply:
Let's move with breaking changes.